### DISCUSSION TITLE
Mark PR as Merged from Pull Request API

### DISCUSSION THREAD

**Discussion by TheShermanTanker** [community/community#24641] [Apps, API and Webhooks]
<p>A while back <a class="mention" href="/xmo-odoo">@xmo-odoo</a> suggested a change to the Pulls API to allow marking a PR as merged regardless of whether any of its commits actually end up inside the codebase (<a href="https://github.community/t/feature-request-mark-pull-request-as-merged/14194/9">mark pull request as merged</a>) and it was listed as added to a feature request tracker shortly after. It’s been a few years since the issue was brought up, are there any updates on this feature request so far? I know it’s been previously mentioned that a timeline/ETA couldn’t be given, but the original discussion has gone quiet since then, giving no indication as to whether the suggestion was ultimately rejected entirely or is still on an internal tracker somewhere and simply hasn’t been implemented yet</p>

**Comment by TheShermanTanker** [comment]
<p>Comment to bump thread</p>

**Comment by TheShermanTanker** [comment]
<p>Bump, tagging <span class="mention">@mchevy</span> for query</p>

**Comment by kaietc** [comment, marked as answer]
<p>Hey <a class="mention" href="/theshermantanker">@TheShermanTanker</a>  - and thank you for this feedback! I’m not too sure if the OP on the original thread had submitted this request to the new Feedback Discussions forum, but please feel free to resubmit it using the link below!</p>
<p>Please submit your feedback via the GitHub <a href="https://github.com/github/feedback/discussions">Feedback Discussions forum</a> so that our product team can better track your request :slight_smile:.</p>

**Comment by TheShermanTanker** [comment]
<p>Hi <a class="mention" href="/kaietc">@kaietc</a>, thanks for the reply, I’ll resubmit the suggestion from the original thread in that case</p>
<p>Have a great day!</p>

**Comment by TheShermanTanker** [comment]
<p>Feature request is now at <a href="https://github.com/github/feedback/discussions/12437" rel="noopener nofollow ugc">[Feature Request] Mark PR as Merged from Pull Request API · Discussion #12437 · github/feedback</a> if anyone is curious</p>

**Comment by TheShermanTanker** [comment]
@kaietc Sorry to trouble you, but is there a way to raise my linked feature request to the attention of the GitHub team? It's been quite a while and I don't wish to see the suggestion completely forgotten and lost to time

**Comment by TheShermanTanker** [comment]
Bumping
